includes/orders/classes/class-order.php

```diff
@@ -446,12 +447,7 @@ public function get_fees() {
 		// Fetch the fees that applied to the entire order.
 		foreach ( $this->adjustments as $adjustment ) {
 			/** @var Order_Adjustment $adjustment */
-
-			if ( 'fee' === $adjustment->type ) {
-				$fee_id = edd_get_order_adjustment_meta( $adjustment->id, 'fee_id', true );
-
-				$fees[ $fee_id ] = $adjustment;
-			}
+			$fees[ $adjustment->type_key ] = $adjustment;
```

Comment:
Sorry I should have caught this earlier.. But I think we should add in some "just in case" checks to see if `$fees[ $adjustment->type_key ]` is empty or if it already exists in the `$fees` array before setting this. `type_key` can technically be null (and would be for manually created orders with manually added fees) and we want to make sure we're not overwriting any data in the array.

If `type_key` is empty, we could fall back to using `$adjustment->id` instead.

This would need to be updated any time we're looping over fees and adding to a `$fees` array.

Reply:
Thanks, I'm glad you caught that--I was actually wondering about that a little since we removed the `type_key` from the manual orders. Will update!